routes/tasks.js

```diff
@@ -0,0 +1,196 @@
+const express = require('express');
+const app = require('../app');
```

Comment:
I don't think you should need this import, since all the uses should be req.app

Not 100% sure though

Reply:
Ahh good catch, will remove